servers/quarkus-server/src/test/java/org/projectnessie/server/ITNativeAuth.java

```diff
@@ -18,4 +18,4 @@
 import io.quarkus.test.junit.NativeImageTest;
 
 @NativeImageTest
-public class ITNativeAuth extends TestAuth {}
+public class ITNativeAuth extends TestBasicOperations {}
```

Comment:
Not renamed?

Reply:
Nice catch :+1:

Reply:
Renamed in 6cce22296722b5f60dfd79b5c4b929de122913e6